packages/z3/z3.4.13.2/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "weng@cs.jhu.edu"
+authors: "MSR"
+homepage: "https://github.com/Z3prover/z3"
+bug-reports: "https://github.com/Z3prover/z3/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Z3prover/z3.git"
+depexts: [
+  ["python3-distutils"] {os-family = "debian"}
+]
+patches: [
+  "gccstd-2a.patch" { (os = "opensuse") | (os-distribution = "ubuntu" & os-version < "20.04") }
```

Comment:
```suggestion
  "gccstd-2a.patch" { (os-family = "opensuse") | (os-distribution = "ubuntu" & os-version < "20.04") }
```

Reply:
From https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values

Reply:
Awesome, thanks for the pointer!